cpu/esp32/include/sdk_conf.h

```diff
@@ -57,11 +57,11 @@ extern "C" {
 /**
  * ESP32 specific configuration
  *
- * CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ can be overridden by an application
+ * CLOCK_CORECLOCK can be overridden by an application
  * specific SDK configuration file.
  */
-#ifndef CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ
-#define CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ       80
+#ifndef CLOCK_CORECLOCK
+#define CLOCK_CORECLOCK       80
```

Comment:
`CLOCK_CORECLOCK` is usually in Hz, so this would be *more* inconsistent.

Reply:
Right, I changed to its all HZ